<launch>
	<machine name="inhands" address="192.168.2.245" env-loader="~/inhands_ws/env.bash" user="ros" password="$(env INHANDS_PSW)"/>

	<arg name="marker_size" default="4.4" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />

	<arg name="cam_image_topic" default="/cam1/qhd/points" />
	<arg name="cam_info_topic" default="/cam1/qhd/camera_info" />		
	<arg name="output_frame" default="/cam1_link" />

	<node pkg="ar_track_alvar" name="ar_track_alvar" type="individualMarkers" output="log" respawn="false" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />
	<node pkg="objects_tracker" name="pose_calibration" type="pose_calibration" output="screen" />
</launch>